src/low-preview/ci.service.js

```diff
@@ -157,6 +161,26 @@ export default class CIResponsive {
     if (!isUpdate) {
       initImageClasses({ imgNode, lazy });
 
+      if (ciZoom) {
+        const zoomIcon = createIcon('../src/common/assets/zoom-icon.png', 'ci-zoom-icon');
+        const modal = createModal(imgSelector, imgNodeSRC);
+        const closeIcon = createIcon('../src/common/assets/close-icon.png', 'ci-close-icon');
+        wrapper.append(zoomIcon);
+        modal.append(closeIcon);
+
+        const openModal = () => {
+          document.body.append(modal);
+          this.process(false, modal);
+        }
+
+        const closeModal = () => {
+          modal.remove();
```

Comment:
remove is not supported in IE, search for alternatives 
https://caniuse.com/?search=remove